### PR TITLE
chore: Azure Stack Hub templates for v0.73.0 disables unattended upgrades

### PR DESCRIPTION
**Pull request by jadarsie**
**Reason for Change**:

There are [reports](https://bugs.launchpad.net/ubuntu/+source/containerd/+bug/1996678) suggesting that upgrading to the Linux kernel version `5.4.0-1095-azure` may cause issues with `containerd`. AKS Engine's [base OS images](https://github.com/Azure/aks-engine/blob/master/vhd/release-notes/aks-engine-ubuntu-1804/aks-engine-ubuntu-1804-202112_2022.11.02.txt#L118) do not include this kernel version. Instructions on how to disable automatic upgrades until a fix is available can be found [here](/docs/howto/troubleshooting.md#prevent-unattended-upgrades).

**Issue Fixed**:

**Credit Where Due**:

Does this change contain code from or inspired by another project?

- [ ] No
- [ ] Yes

If "Yes," did you notify that project's maintainers and provide attribution?

- [ ] No
- [ ] Yes

**Requirements**:

- [ ] uses [conventional commit messages](https://www.conventionalcommits.org/)
  
- [ ] includes documentation
- [ ] adds unit tests
- [ ] tested upgrade from previous version

**Notes**:

### CODE REVIEW

File: docs/topics/azure-stack.md [hunk 1]
```diff
@@ -156,10 +156,12 @@ Make sure `linuxProfile.runUnattendedUpgradesOnBootstrap` is set to `"false"` wh
...
 
 >\* Starting from Kubernetes v1.21, **only** [Cloud Provider for Azure](#cloud-provider-for-azure) is supported on Azure Stack Hub.
 
+>\** There are [reports](https://bugs.launchpad.net/ubuntu/+source/containerd/+bug/1996678) suggesting that upgrading to the Linux kernel version `5.4.0-1094-azure` may cause issues with `containerd`. AKS Engine's base OS images do not include this kernel version. Instructions on how to disable automatic upgrades until a fix is available can be found [here](/docs/howto/troubleshooting.md#prevent-unattended-upgrades).
```
Comment (christinalau0) [inline docs/topics/azure-stack.md:163]: Should be 5.4.0-1095? AKS Engine base OS image does include 5.4.0-1094-azure

**Comment by christinalau0** [comment]
Small typo in title, v0.73.0

File: docs/topics/azure-stack.md (hunk 1 above)
Comment (jadarsie) [reply docs/topics/azure-stack.md:163]: yes, thx

**Review by christinalau0 — APPROVED** [review]
lgtm

**Pull request merged**